### ISSUE TITLE
Boxmenu is using linkText for button aria-label, needs a properly named attribute

### ISSUE THREAD

**Issue by dancgray** [adaptlearning/adapt_framework#1676]
Using linkText [here](https://github.com/adaptlearning/adapt-contrib-boxmenu/blob/master/templates/boxmenu-item.hbs#L14) means that the menu button aria label cannot be edited in the authoring tool.

Should it be _globals._accessibility._ariaLabels.menuViewButton?

**Comment by moloko** [comment]
surely that would only be necessary if you wanted the aria label to be different to the button label

**Comment by dancgray** [comment]
Indeed. I raise the issue in response to a question on gitter from @mldaq.

If the consensus is that for accessibility they should be the same feel free to close the issue.

**Comment by mldaq** [comment]
This came up when an accessible user was trying to navigate our course and came to the "View" button - JAWS stated the same, but the user was not sure what that button was going to do for him. I was thinking it would be nice to provide more context for accessible users, like - "Select to learn more" or "Select to open this section". I like View as the button text, and would not want to change that for non-accessible folks.

**Comment by oliverfoster** [comment]
@dancgray it shouldn't be in the globals because it's content specific and will vary for each contentObject. a new property should be included on each menu item if this is still required.

**Comment by moloko** [comment]
see https://github.com/adaptlearning/adapt-contrib-boxmenu/pull/83

**Comment by oliverfoster** [comment]
This should be clearer with the buttons appearing as a list. The context is much richer now. @mldaq can I consider this closed?
